### ISSUE TITLE
[Feature Request] Add system wide font and spacing control to redactor

### ISSUE THREAD

**Issue by danielw0581** [osTicket/osTicket#5262]
It would be great to have a system wide setting that controls the font and formatting in redactor. As of the update to 1.14.1 (Previously we were on 1.12.1) there is double spacing in redactor and we have had to revert back to 1.12.1 in order to maintain single spacing in emails. The new spacing makes the emails looks terrible.

We also got stuck in overtype mode in redactor when using 1.12.1.

Redactor seems to need a lot of work.

**Comment by TechCare-Aus** [comment]
I agree. To prevent double line you need to hold down Shift + Enter. But that is annoying.

You can follow the redactor commits discussion here - https://github.com/osTicket/osTicket/pull/4690
